### PR TITLE
mg25x

### PR DESCRIPTION
**Pull request by pmandrik**
Switch MG_amc@nlo gridpack generation from v2.5.1 to v2.5.4 and update related patches.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/gridpack_generation.sh [hunk 1]
```diff
@@ -119,8 +119,8 @@ if [ ! -d ${AFS_GEN_FOLDER}/${name}_gridpack ]; then
 #   export SCRAM_ARCH=slc6_amd64_gcc472 #Here one should select the correct architechture corresponding with the CMSSW release
 #   export RELEASE=CMSSW_5_3_32_patch3
 
-  export SCRAM_ARCH=slc6_amd64_gcc481
-  export RELEASE=CMSSW_7_1_23
+  export SCRAM_ARCH=slc6_amd64_gcc530
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/gridpack_generation.sh]: ah, so is intended that we switch to gcc5 also for MG5_aMC@NLO. This is an even stronger case to urgently discuss with computing how to handle the production of MC events for 2016 data analysis in the future...

File: bin/MadGraph5_aMCatNLO/gridpack_generation.sh [hunk 2]
```diff
@@ -119,8 +119,8 @@ if [ ! -d ${AFS_GEN_FOLDER}/${name}_gridpack ]; then
...
+  export RELEASE=CMSSW_9_0_2
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/gridpack_generation.sh]: is there a specific reason why 902 is used and not, for instance 920?

**Comment by perrozzi** [comment]
things look ok to me, let's wait for some other comment

**Comment by bendavid** [comment]
Is there a particular reason to change the default release and scram_arch?

**Comment by pmandrik** [comment]
Dear all, there is no specific reason of release and scram_arch changes.
It was made in order to try to bypass the bug in 2.5.1-2.5.2, didn't help and left in the code.
Are there any reasons to prefer previous versions?

**Comment by bendavid** [comment]
Yes, 71x and the corresponding scram_arch is strongly preferred to maintain compatibility with existing campaigns.

**Comment by perrozzi** [comment]
I assume is ok to merge now

**Pull request merged**